bin/karma

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+bin/checkinputs "$@"
+
+export OX_PROJECT=$1
+echo Karma: $OX_PROJECT
+
+karma start test/karma.config.js --auto-watch --no-single-run
```

Comment:
now that we have this, i wonder if we should just call this from with the `bin/tdd`?